proofs/hol_light/arm/README.md

```diff
@@ -13,6 +13,22 @@ proved. Specifications are essentially Hoare triples, with the noteworthy differ
 that the program is implicit as the content of memory at the PC; which is asserted to
 be the code under verification as part of the precondition.
 
+## What is covered?
+
+At present, this directory contains functional correctness proofs for the following functions:
+
+- ML-KEM Arithmetic:
+  * Optimized AArch64 forward NTT: [mlkem_ntt.S](mlkem/mlkem_ntt.S)
+  * Optimized AArch64 inverse NTT: [mlkem_intt.S](mlkem/mlkem_intt.S)
+- FIPS202:
+  * Keccak-F1600 using lazy rotations (see [this paper](https://eprint.iacr.org/2022/1243)): [keccak_f1600_x1_scalar.S](mlkem/keccak_f1600_x1_scalar.S)
+  * Keccak-F1600 using v8.4-A SHA3 instructions: [keccak_f1600_x1_v84a.S](mlkem/keccak_f1600_x1_v84a.S)
+  * 2-fold Keccak-F1600 using v8.4-A SHA3 instructions: [keccak_f1600_x2_v84a.S](mlkem/keccak_f1600_x2_v84a.S)
+  * 'Hybrid' 4-fold Keccak-F1600 using scalar and v8-A Neon instructions: [keccak_f1600_x4_v8a_scalar.S](mlkem/keccak_f1600_x4_v8a_scalar.S)
+  * 'Triple hybrid' 4-fold Keccak-F1600 using scalar, v8-A Neon and v8.4-A+SHA3 Neon instructions:[keccak_f1600_x4_v8a_v84a_scalar.S](mlkem/keccak_f1600_x4_v8a_v84a_scalar.S)
```

Comment:
Can you add this list to the root README as well? Looks nicer than the one we have there so far.

Reply:
Done